src/ui-test/variables.ts

```diff
@@ -4,6 +4,9 @@ export const TEST_RESOURCES_DIR = path.resolve('.', 'test-resources');
 export const EXTENSION_DIR = path.join(TEST_RESOURCES_DIR, 'test-extensions');
 export const WORKBENCH_DIR = path.join(TEST_RESOURCES_DIR, 'ui-workbench');
 export const RESOURCES_DIR = path.resolve('.', 'src', 'ui-test', 'resources');
+export const RESOURCES_TASK_EXAMPLES_DIR: string = path.resolve(RESOURCES_DIR, 'tasks-examples');
```

Comment:
it is just small optimisation, in this case you don't need to use `path.resolve` again, just simply use only `path.join`, the full path you need was already once resolved for `RESOURCES_DIR `